restapi/tenants.go

```diff
@@ -63,6 +63,30 @@ func registerTenantHandlers(api *operations.M3API) {
 		return admin_api.NewTenantInfoOK().WithPayload(resp)
 
 	})
+
+	// Detail Tenant
+	api.AdminAPIDeleteTenantHandler = admin_api.DeleteTenantHandlerFunc(func(params admin_api.DeleteTenantParams) middleware.Responder {
+		err := getDeleteTenantResponse(params)
+		if err != nil {
+			return admin_api.NewTenantInfoDefault(500).WithPayload(&models.Error{Code: 500, Message: swag.String(err.Error())})
+		}
+		return admin_api.NewTenantInfoOK()
+
+	})
+}
+
+func getDeleteTenantResponse(params admin_api.DeleteTenantParams) error {
+	opClient, err := operatorClientset.NewForConfig(cluster.GetK8sConfig())
+	if err != nil {
+		return err
+	}
+	currentNamespace := cluster.GetNs()
+
+	err = opClient.OperatorV1().MinIOInstances(currentNamespace).Delete(context.Background(), params.Name, metav1.DeleteOptions{})
```

Comment:
can you implement this function call inside the `restapi/operator_client.go` so we can use it in future testing.